tests/performance_tests/log_reader.py

```diff
@@ -198,7 +198,26 @@ def scoreTransfersPerSecond(data: chainData, guide : chainBlocksGuide) -> stats:
         # Note: numpy array slicing in use -> [:,0] -> from all elements return index 0
         return stats(int(np.min(npCBTAEC[:,0])), int(np.max(npCBTAEC[:,0])), float(np.average(npCBTAEC[:,0])), float(np.std(npCBTAEC[:,0])), int(np.sum(npCBTAEC[:,1])), len(prunedBlockDataLog))
 
-def createJSONReport(guide: chainBlocksGuide, tpsStats: stats, args) -> json:
+def calcBlockSizeStats(data: chainData, guide : chainBlocksGuide) -> stats:
+    """Analyzes a test scenario's steady state block data for statistics blocks size during the test window"""
```

Comment:
"for block size statistics" perhaps?

Reply:
Resolved: 1c6030e834d1b4f84c483536bb8ab651b8c7defb